--- old/generic.py	2016-05-19 01:47:24.000000000 +0900
+++ new/generic.py	2017-10-03 09:43:12.318495974 +0900
@@ -483,6 +483,10 @@
         try:
             return NameObject(name.decode('utf-8'))
         except (UnicodeEncodeError, UnicodeDecodeError) as e:
+            try:
+                return NameObject(name.decode('cp932'))
+            except:
+                pass
             # Name objects should represent irregular characters
             # with a '#' followed by the symbol's hex number
             if not pdf.strict:
